juju/juju
# Fixes error with ensure ClusterRole

When ensuring a cluster role if the labels had changed then update would
fail with an already exists error. As reported in lp1929909.

## Checklist


## QA steps

Upgrade from 2.8 to 2.9 or use regression test added with this PR. See the bug below for more information.

## Bug reference

https://bugs.launchpad.net/juju/+bug/1929909
